docs/docs/en/guide/project/workflow-definition.md

```diff
@@ -74,14 +74,22 @@ The following are the operation functions of the workflow definition list:
     * Recipient: select notification policy||timeout alarm||when fault tolerance occurs, process result information or alarm email will be sent to the recipient list
     * Cc: select notification policy||timeout alarm||when fault tolerance occurs, the process result information or warning email will be copied to the CC list
     * Startup parameter: Set or overwrite global parameter values when starting a new process instance
-    * Complement: two modes including serial complement and parallel complement. Serial complement: within the specified time range, the complements are executed from the start date to the end date and N process instances are generated in turn; parallel complement: within the specified time range, multiple days are complemented at the same time to generate N process instances.
+    * Complement: two modes including serial complement and parallel complement. Serial complement: within the specified time range, the complements are executed from the start date to the end date and N process instances are generated in turn; parallel complement: within the specified time range, multiple days are complemented at the same time to generate N process instances. Manual input, serial complement: manually enter the date in comma separated date format as `yyyy-MM-dd HH:mm:ss`, and generate n process instances in turn; Parallel Replenishment: manually enter a date in the comma separated date format of `yyyy-MM-dd HH:mm:ss`. Complement data simultaneously for multiple days and generate N process instances at the same time.
```

Comment:
* Complement: use date selection or manual input to make complement, and each of these methods contains serial complement and parallel complement. Serial complement defines, within the specified time range, executes the complements from the start date to the end date by sequential and N process instances are generated in sequence. Parallel complement defines executing the complement concurrently in the multiple days to generate N process instances within the specified time range. Manual date input defines: manually input the date following the date format `yyyy-MM-dd HH:mm:ss` and separate dates by a comma. Date selection defines: to select dates via UI.